android/src/main/java/org/mozilla/mozstumbler/client/serialize/ObservationPointSerializer.java

```diff
@@ -52,9 +52,10 @@
     private static final String STYLE_NAME_RED_CIRCLE = "redcircle";
     private static final String ICON_ARROW = "http://earth.google.com/images/kml-icons/track-directional/track-0.png";
     private static final String STYLE_NAME_ARROW = "arrow";
-    private static final String COLOR_HAS_WIFI = "ffff0000"; //green in AABBGGRR
-    private static final String COLOR_HAS_CELLS = "ff00ffff"; // yellow
-    private static final String COLOR_HAS_BOTH = "ff00ff00"; // blue
+    private static final String COLOR_HAS_WIFI = "ff0000ff"; // red in AABBGGRR
+    private static final String COLOR_HAS_CELLS = "ffff0000"; // blue
+    private static final String COLOR_HAS_BOTH = "ff00ff00"; // green
+    private static final String COLOR_HAS_NONE = "aa000000"; // transparent black
```

Comment:
This changes the colors of the on-map points, right? Do we have a legend for these that would need to get updated? Or should we make a note to mention this in the news section for the next play store release?